solarwinds_apm/configurator.py

```diff
@@ -384,8 +386,21 @@ def _configure_metrics_exporter(
         self,
         apm_config: SolarWindsApmConfig,
     ) -> None:
-        """Configure SolarWinds OTel metrics exporters if any configured.
-        Links them to new metric readers and global MeterProvider."""
+        """Configure OTel OTLP metrics exporters if enabled and agent enabled.
+        Settings precedence: OTEL_* > SW_APM_EXPORT_METRICS_ENABLED.
+        Links to new metric readers and global MeterProvider."""
+        if not apm_config.agent_enabled:
+            logger.error(
+                "APM Python library disabled. Cannot set metrics exporters."
+            )
+            return
```

Comment:
This was previously missing for `_configure_metrics_exporter`, so I've added it now.